### ISSUE TITLE
 add namespacing to $.fn.drag

### ISSUE THREAD

**Issue by jurgenkalverboer** [ducksboard/gridster.js#499]
I use another library that implements the $.fn.drag funtion. The implementation of that library should globally be available where the implementation in gridster is never used outside gridster. Thus could you add some namespacing to this function to make it less generic

**Comment by triplethreat77** [comment]
Agreed. I am pairing Gridster with fusioncharts and when a chart block loads, the drag is broken on the gridster.
